### PR TITLE
Fixed some typos and grammar in README.md

### PR DESCRIPTION
**Pull request by asridharan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 # DC/OS Tools, Packages, and Installers for various platforms
 
-Tools for building DC/OS and launchign a cluster with it in the hardware of a customer's choice.
+Tools for building DC/OS and launching a cluster with it on a hardware of user's choice.
 
-  - *docker/* Locally defined docker containers packages are built in
+  - *docker/* Locally defined docker containers packages are built
```
Comment (cmaloney) [inline README.md]: we don't have a hard wrap limit in the markdown / this shouldn't wrap.

Also note the dockers aren't actually built in this folder. The folder simply contains Dockerfile definitions. Each dockerfile is in a folder which has the name that the particular builder should use (ex: dcos-builder). When people say "docker": "dcos-builder" these dockers will get lazily built when requested. They'll then be used to build the package which asked for that particular docker as a build environment.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+    here
   - *docs/* Documentation
-  - *ext/dcos-installer/* Backend for Web, SSH, and some bits of the Advanced installer. To be merged into the top codebase once the code is cleaned up
-  - *gen/* Python library for rendering yaml config files for various platforms into packages, with utilities to do things like make "late binding" config set by CloudFormation
+  - *ext/dcos-installer/* Backend for Web, SSH, and some bits of the
+    *Advanced Installer*.
```
Comment (cmaloney) [inline README.md]: The note that this `ext/` folder is supposed to be temporary I believe is valuable

**Comment by cmaloney** [comment]
I don't agree with most of the changes here / they aren't technically correct in describing the codebase. I left comments on some of them. Closing this PR for now.

**Comment by asridharan** [comment]
Sure,
 Probably should have raised an #issue rather than a submitted a PR.
Basically I feel the text is a bit too cryptic for people understand the
build-tree structure. This I believe is essential for developers and
contributors to get familiar with the DC/OS code-base.

On Thu, Apr 21, 2016 at 1:16 PM, Cody Maloney <notifications@github.com>
wrote:

> I don't agree with most of the changes here / they aren't technically
> correct in describing the codebase. I left comments on some of them.
> Closing this PR for now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dcos/dcos/pull/4#issuecomment-213098037>
>



-- 
Avinash Sridharan, Mesosphere
+1 (323) 702 5245

**Pull request closed without merging**